src/playbook.yaml

```diff
@@ -2,6 +2,33 @@
   become: yes
 
   tasks:
-  - name: Print debug message
-    debug:
-      msg: Hello, world!
+
+  - name: Add Docker GPG key
+    apt_key: url=https://download.docker.com/linux/ubuntu/gpg
+
+  - name: Add Docker APT repository
+    apt_repository:
+      repo: deb [arch=amd64] https://download.docker.com/linux/{{ ansible_distribution|lower }} {{ansible_distribution_release}} stable
+      update_cache: yes
+
+  - name: install packages
+    package:
+      name:
+        - python3-pip
+        - docker-ce
+        - docker-ce-cli
+        - containerd.io
+      state: present
+
+  - name: Docker post-install
```

Comment:
I think you'll additionally need a post-install line to run containerd on boot, like a 'systemctl enable dockerd/containerd' or some such.

Reply:
docker and containerd are set up to start on boot through systemd

Reply:
Oh sorry- I was thinking more of if you needed to start a specific container n/m